### ISSUE TITLE
cool tech, can you provide use case

### ISSUE THREAD

**Issue by freeslugs** [flouthoc/buha.js#1]
can you provide real-life example of why one might want to use this? 

why push sync function using buha vs. just running sync function normally?

**Comment by flouthoc** [comment]
@freeslugs  Yes sure if its only sync you dont need a task queue but if your events are combinations of async / sync event or only async events and you want them to run in a chronological order you can use this task queue

**Comment by germangamboa95** [comment]
I think it would be cool to see this off load the queue into a service worker. Might I submit a PR for that?

**Comment by flouthoc** [comment]
@germangamboa95 Sure !

**Comment by germangamboa95** [comment]
Awesome I'll start work on that.

**Comment by flouthoc** [comment]
:+1:
